quiz/envs/postgres.default.env

```diff
@@ -1,7 +1,7 @@
 POSTGRES_USER=
 POSTGRES_PASSWORD=
 POSTGRES_DB=
-POSTGRES_HOST=
+POSTGRES_HOST=postgres
 POSTGRES_PORT=
```

Comment:
I think we can also add a default port for postgres 5432

Reply:
added